### PR TITLE
CHE-2888: Fix pull requests-build due to git tests.

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
### What does this PR do?

 Fix pull requests-build due to git tests
### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/2888

Signed-off-by: Aleksandr Andrienko aandrienko@codenvy.com

### CODE REVIEW

**Review by vinokurig — APPROVED** [review]

**Comment by vinokurig** [comment]
ci-build

File: wsagent/che-core-git-impl-jgit/pom.xml [hunk 1]
```diff
@@ -129,17 +129,17 @@
...
-            <scope>test</scope>
-        </dependency>
-        <dependency>
             <groupId>org.eclipse.che.core</groupId>
             <artifactId>che-core-api-git</artifactId>
             <classifier>tests</classifier>
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>org.hamcrest</groupId>
```
[the remark this answers is not in the document]
Comment (AndrienkoAleksandr) [reply wsagent/che-core-git-impl-jgit/pom.xml]: MockitoTestNGListener

[the remark this answers is not in the document]
Comment (sleshchenko) [reply wsagent/che-core-git-impl-jgit/pom.xml]: MockitoTestNGListener is from mockitong, no?

[the remark this answers is not in the document]
Comment (AndrienkoAleksandr) [reply wsagent/che-core-git-impl-jgit/pom.xml]: Yes

**Comment by AndrienkoAleksandr** [comment]
ci-build

**Review by skabashnyuk — APPROVED** [review]

**Pull request merged**